src/main/java/it/gov/pagopa/payhub/activities/service/classifications/TransferClassificationService.java

```diff
@@ -38,14 +40,16 @@ public TransferClassificationService(List<TransferClassifier> classifiers) {
 	 * @param transferDTO the transfer data.
 	 * @param paymentsReportingDTO the payment reporting data.
 	 * @param treasuryIuf the treasury IUF data.
+	 * @param installmentDTO the installment data.
 	 * @return a list of valid labels generated by the classifiers.
 	 */
 	public List<ClassificationsEnum> defineLabels(Transfer transferDTO,
 	                                              PaymentNotificationNoPII paymentNotificationDTO,
 	                                              PaymentsReporting paymentsReportingDTO,
-	                                              TreasuryIuf treasuryIuf) {
+	                                              TreasuryIuf treasuryIuf,
+	                                              Optional<InstallmentNoPII> installmentDTO) {
```

Comment:
don't use Optional, ask directly for the Object, as done for the other parameters

ps: semantically, it could be more readable to have it as 2nd parameter, right after the Transfer itself, due to the strict relationship between them (ps: if a Transfer exists, also the Installment exists! (transfer has a foreign key towards installment)
```suggestion
	                                             InstallmentNoPII installmentDTO) {
```